README.md

```diff
@@ -62,10 +62,20 @@ brew install poetry
 
 ## Common Development Setup
 
-1. Install Python & Poetry (above)
-2. Clone the [GG2 Github repo](https://github.com/DistributedProofreaders/guiguts-py)
+1. Install Python, Poetry, etc.
+   * Example from Ubuntu 22.04 -- adapt to your own Linux distro
+     ```bash
```

Comment:
These are Linux-specific so we should add them to a `Linux Development Setup` section (similar to what we did with the `macOS Development Setup`) and not here in the common bits.